src/mlpack/core/optimizers/cmaes/random.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <ctime>
+#include <cmath>
+
+/**
+ * @class Random
+ * A pseudo random number generator.
+ */
+template<typename T>
+class Random
```

Comment:
If you just need a single random number you can use ``RandNormal`` or ``Random`` from ``core/math/random.hpp``. But I think ``arma::as_scalar(arma::randu(1))`` should also work, can you elaborate on the problem, you encountered?